src/components/Category/components/Image.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const StyledImage = styled.img`
+    width:100%;
+    height:auto;
+    max-height: 300px;
+    margin: 0;
+    padding:0;
+
+`;
```

Comment:
ten obrazek domyślam się że jest tak tylko testowo, ale ogólnie można ogarnąć na przyszłość żeby miał stosunek 16:9

Reply:
Tak to testowy, żeby było cokolwiek